environment/commands/stop/all.js

```diff
@@ -0,0 +1,30 @@
+const { StopLocalNearNodeCommand } = require('./near.js');
+const { StopManagedProcessCommand } = require('./process.js');
+
+class A {
+}
+
+stop = function(name) {
+    try {
+        var a = A;
+        a._name = name;
+        StopManagedProcessCommand.execute(a);
+    } catch (err) {
+        console.log('Error stopping', name, err);
+    }
+}
+
```

Comment:
Please suggest how to make it better.

Reply:
I think this is fine. For future housekeeping, refactor StopMangedProcessCommand to a function stop(name), and in here and StopManagedProcessCommand, call stop(name)